docs/user-guide/networkpolicies.md

```diff
@@ -0,0 +1,82 @@
+---
+---
+
+* TOC
+{:toc}
+
+## What is a _Network Policy_?
+
+A Network Policy is a specification of how groupings of pods are allowed to communicate with each other and other network endpoints.
```

Comment:
QQ: Is there a name for the mechanism that groups these pods? Is it a namespace? If so say so, "groupings" is a little vague.